core/src/main/java/nl/nn/adapterframework/webcontrol/api/ShowConfigurationStatus.java

```diff
@@ -719,15 +719,17 @@ private Map<String, Object> mapAdapter(Adapter adapter) {
 				try {
 					errorStoreMessageCount += esmb.getMessageCount();
 				} catch (ListenerException e) {
```

Comment:
```suggestion
				} catch (ListenerException e) {
						// Only log the stacktrace when loglevel == INFO. Otherwise it will pollute the log too much.
```